build.cake

```diff
@@ -48,6 +51,7 @@ var semVersion = version + (isLocal ? string.Empty : string.Concat("-build-", bu
 var buildDir = Directory("./src/clients/Wyam/bin") + Directory(configuration);
 var buildResultDir = Directory("./build") + Directory(semVersion);
 var nugetRoot = buildResultDir + Directory("nuget");
+var chocoRoot = buildResultsDir + Directory("choco");
```

Comment:
@phillipsj the AppVeyor build is currently failing.  I think the issue is here... `buildResultsDir` -> `buildResultDir`